src/core/stdc/stdint.d

```diff
@@ -45,23 +61,6 @@ static if (is(ucent))
 
 version (Windows)
 {
-    alias int8_t   = byte;   ///
-    alias int16_t  = short;  ///
-    alias uint8_t  = ubyte;  ///
-    alias uint16_t = ushort; ///
-    version (CRuntime_DigitalMars)
-    {
-        alias int32_t  = cpp_long;  ///
-        alias uint32_t = cpp_ulong; ///
```

Comment:
`cpp_long` and friends have special mangling so these will have to remain I think

Reply:
Ouch, totally forgot that. I was in the C world forgetting C++ considerations.